src/Components/Web/src/Forms/InputBase.cs

```diff
@@ -25,7 +26,7 @@ public abstract class InputBase<TValue> : ComponentBase, IDisposable
         /// <summary>
         /// Gets or sets a collection of additional attributes that will be applied to the created element.
         /// </summary>
-        [Parameter(CaptureUnmatchedValues = true)] public IReadOnlyDictionary<string, object> AdditionalAttributes { get; set; }
+        [Parameter(CaptureUnmatchedValues = true)] public Dictionary<string, object> AdditionalAttributes { get; set; }
```

Comment:
Out of interest, is `IDictionary` also a viable option here?

Reply:
Dictionary implements `IReadOnlyDictionary` and is not a code breaking change, where as if I change this to `IDictionary` instead, tests where we have something like the following, break. I expect this will also break customer code.

```csharp
public new IReadOnlyDictionary<string, object> AdditionalAttributes => base.AdditionalAttributes;
```

Let me know if you want me to change this to IDictionary - I am not against it.